### PR TITLE
Master universal uom jem

### PR DESCRIPTION
**Pull request by jem-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/uom/data/uom_data.xml [hunk 1]
```diff
@@ -39,28 +43,48 @@
...
         </record>
         <record id="product_uom_gram" model="uom.uom">
             <field name="category_id" ref="product_uom_categ_kgm"/>
             <field name="name">g</field>
             <field name="factor" eval="1000"/>
             <field name="uom_type">smaller</field>
+            <field name="international_unit">gram</field>
+        </record>
+
+        <record id="product_uom_time_day" model="uom.uom">
```
Comment (Feyensv) [inline addons/uom/data/uom_data.xml]: Why product_uom_time and not time_uom or uom_time ?
just because of the naming of other xml ids? :D

Comment (jem-odoo) [reply addons/uom/data/uom_data.xml]: Yes, but it is true: 'uom_time_xxx' is better !

**Comment by jem-odoo** [comment]
As @dbo-odoo is ok with that, let's merge this !
@robodoo rebase-ff r+

**Comment by jem-odoo** [comment]
@robodoo r+

**Pull request closed without merging**